Examples/ByFeature/CustomContainer/paket.references

```diff
@@ -1,4 +1,6 @@
 ﻿Autofac
 FSharp.Core
 unquote
-Xunit
\ No newline at end of file
+Xunit
+xunit.runner.visualstudio
+Microsoft.NET.Test.Sdk
```

Comment:
Why this is needed?

Reply:
It is package that is added byt default to .NET Core test projects, see e.g. https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-nunit

True is that I don't know full details, I know that I had some issues without it.

There was an issue https://github.com/Microsoft/vstest/issues/986 which contains partially reason why it is needed but also try to avoid need for it for some cases.

Reply:
Sounds good, thanks.